data/remnant/remnant logistics.txt

```diff
@@ -0,0 +1,295 @@
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Remnant: Logistics 1"
+	name "Cargo delivery to Viminal"
+	description `Deliver <cargo> to <destination>.`
+	source "Caelian"
+	destination "Viminal"
+	cargo "Project ID: 87FA9032B" 30
+	to offer
+		has "remnant: blood test pure"
+		has "Remnant: Defense 1: done"
+		has "cargo space" >= 30
+	on offer
+		conversation
+			`You step out of the <ship> and look again at the strange sight of the Caelian spaceport. You still aren't used to a structure that looks as much like it grew from the landing pads as it was built next to them. The dock side entrance is busy so when your commlink vibrates, you have to find a place where the traffic eddies and you can step aside and learn that you need to check in with the Information Desk.`			
+			`	The woman working the desk seems startled to see you when you walk up. You can see the realization dawn on her as she scans your clothes. "Ah," she sings, "you are the ancestral sojourner among us."`
+			`	You are getting used to being gawked at. So, you take it in good humor and respond by singing, "The Embers guided me to friends I didn't know, and your message invited me to meet a new one."`
+			`	Her hands make a short series of shakes and a moment later she remembers to laugh for you. The delayed response makes her laugh again, spontaneously this time, and you both join in. When she recovers herself, she sings, "Logistics have requested that we find someone who can carry 30 tons of cargo to Viminal. Is the <ship> available immediately?"`
+			choice
+				`	(Accept the job)`
+					goto willdo
+				`	(I'm not interested in small cargo jobs)`
+					goto notinterested
+			label willdo
+			`	"Certainly," you chant, curious now as the job board has been stubbornly empty when you've checked.`
+			`	"Excellent," she responds in a full alto, "I will let the project manager know the cargo will arrive tomorrow." She signs what you think is a farewell and turns to sign to someone else who has stepped up.`
+			`	You manage to chant and point your way through the cafeteria line and you are aware of the flurry of signs around you. There doesn't seem to be a way to pay, so you find a table and sit alone until your commlink chimes to let you know that the <ship> has been refueled and the mystery cargo loaded.`
+				accept
+			label notinterested
+			`	"That is unfortunate," she sings as she makes a notation on her terminal. "But there will be others who are willing to help."`
+				decline
+	on complete
+		payment 50000
```

Comment:
I've been giving this some thought and I think the Remnant would pay more than human space because of their structure and limited resources (e.g keystones), but there needs to be a wider spread, with salvage being much more valuable. One issue is that raider salvage is so lucrative everything else looks inconsequential by comparison. Eon's conversation should set the expectation that the jobs are fill in work, so I need to revisit that as well. Thank you for doing the comparison. I was focused on just getting the jobs to run in the first instance and this helps me quite a bit.